sites/cheerpx/src/content/docs/11-guides/Networking.md

```diff
@@ -22,21 +22,51 @@ Native networking in the browser is extremely limited, only allowing HTTP connec
 > 2. The browser always enforces cross-origin rules and cross-domain requests are only allowed by a small subset of domains, making it very unlikely that an aribitrary application could work when attempting HTTP traffic via fetch.
 >    These limitations necessitate a more comprehensive networking solution that can provide a full TCP/IP stack.
 
-#### The need for a VPN solution
+### The need for a VPN solution
 
 To overcome these limitations, CheerpX uses Tailscale, which provides a VPN solution via WebSockets. This approach allows CheerpX to implement a complete TCP/IP stack, enabling applications inside the VM to perform networking operations as they would on a regular machine.
 
 Networking with WebVM always happens via Tailscale. To give access to your local or development machine as part of the Tailscale internal network, you need to install Tailscale locally. The setup is effectively the same as with internet-enabled configurations; you can just skip enabling the exit node if you only need internal network access.
 
-It's important to note that WebVM-to-WebVM networking can be achieved via Tailscale without the need for any exit node. This solution could be useful for client/server applications running entirely without WebVM and peer-to-peer applications such as multiplayer games.
+> [!note] Note
+> WebVM-to-WebVM networking can be achieved via Tailscale without the need for any exit node. This solution could be useful for client/server applications running entirely without WebVM and peer-to-peer applications such as multiplayer games.
 
 For a more detailed explanation of why Tailscale was chosen and how it works with WebVM, you can read our [blog post on WebVM networking via Tailscale](https://labs.leaningtech.com/blog/webvm-virtual-machine-with-networking-via-tailscale).
 
-### Client applications with existing backends
+## Exit node
 
-The easiest way to connect a client application running in CheerpX with traditional backends, either on the public internet or private network), is to [install Tailscale](https://tailscale.com/kb/1017/install) on your backend server. This way the client can connect to your private Tailscale network via an auth key, and then talk to your server via the VPN.
+An exit node is a device on the Tailscale network that allows other Tailscale devices to access the internet by routing their traffic through its connection. This makes it possible for traffic to reach the internet.
 
-### Connecting your application to a Tailscale network
+### When do you need an exit node?
+
+You need an exit node if you want to:
+
+- `Access the internet` from devices connected to your Tailscale network.
+- `Route traffic through a trusted device` to improve privacy or security.
```

Comment:
Can you elaborate on the purpose of this line? What is the concern?

Reply:
When reading through the Tailscale Exit nodes page I felt like they are big on their security so that's why I left it in. It might be extra information that's not needed in our docs. Would you like me to remove that part as well?